crates/bevy_ui/src/flex/mod.rs

```diff
@@ -22,6 +24,16 @@ use taffy::{
     Taffy,
 };
 
+#[derive(Component, Copy, Clone)]
+pub struct TaffyNode {
```

Comment:
Could we collapse these two components into `TaffyState` or just merge it into `Node`? Unless there's a good reason (i.e. parallelism, ambiguity, cache bandwidth) to keep these as separate components, I'm not sure it's worth it to introduce two more components.

Reply:
They can't go in `Node` I think because it's queried for size change detection.
Maybe they can be combined into a single component though. They aren't components that users should ever need to be aware of and they aren't part of any bundles.

Reply:
Just realised, I think `TaffyParent` is only required because we're still using 'Transform` and it can be removed when we switch to a UI-specific transform.

Reply:
+1 for combining into a single component. I see no reason for these to be separate and combining them will save a lookup in the case you need both.